src/main/java/net/kyori/adventure/text/minimessage/transformation/inbuild/FontTransformation.java

```diff
@@ -65,17 +65,15 @@ public void load(final String name, final List<TagPart> args) {
     super.load(name, args);
 
     if (args.size() == 1) {
-      @Subst("minecraft:empty") final String fontKey = args.get(0).value();
+      @Subst("empty") final String fontKey = args.get(0).value();
```

Comment:
it better reflects the form of the argument (since its not namespaced if there's only one arg) but at the end of the day it doesn't really matter because it's just to appease the IDE